forecast/paper/response-to-reviewers.tex

```diff
@@ -0,0 +1,304 @@
+  we would expect that different weights may alter the conclusions, though
+  likely not substantively. Applying a monotone function to tau only effectively
+  changes the quantile of interest, resulting in a mismatch between coverage you
+  tried to get and the coverage WIS is evaluating. Applying the function to $\tau$
+  and $(1-\tau)$ results in an asymmetry, penalizing forecasts that miss on one
+  side more than on the other.
+
+  $\bullet$ We have added a sentence in the Evaluation Metrics section emphasizing that
+  this metric is standard.
+\end{resp}
+
+
+  2. They do not discuss the possible effects of vaccination rates and other
+  factors, though this may be one of the areas of localization they propose to
+  investigate.
+
+
+
+\begin{resp}
+  During the period discussed in the main paper, vaccinations hadn’t really
+  taken off. On Dec 31 2000, when the main evaluation period ends,  according to
+  CDC data, about 1\% of the population 
+  had received 1 dose. That said, we should probably be more specific in the
+  paragraph on lines 31-40: our goal is to use very simple models that do not
+  account for extra information (vaccinations but also strains of the virus,
+  ``super-spreader'' events, etc.).
+
+  $\bullet$ We have added a sentence around lines 30--40 emphasizing the choice
+  to ignore such factors.
+
+  $\bullet$ We also added some text to the Prediction Period section stating
+  that our end 
+  date (31 December) coincides with the introduction of vaccinations.
+\end{resp}
+
+
+  3. This points to what is perhaps a more major issue. Can they identify in
+  quantitative terms the consequences of the additional improvements in
+  forecasting provided by these methods in terms of gains in resources available
+  to deal with the epidemic? Unfortunately as they state, in the crucial periods
+  of sharply rising cases, the additional benefit seems least.
+
+
+\begin{resp}
+  This is a good point, and we have made a few modifications described in more
+  detail below (Reviewer 2, Point 3).
+\end{resp}
+
+
+
+  4. Despite the fact that a detailed description of terms in the text is
+  impossible, it would I think be helpful to say if there is any significance to
+  the names they associate with their first 2 and 4th indicators.
+
+
+\begin{resp}
+  It seems that the Reviewer is referring to CHNG-COVID and DV-CLI. These are
+  described in the Methods Section, Signals and Locations Subsection. To be more
+  specific, Change 
+  Healthcare is a large healthcare insurance claims processor. The difference
+  between ``-CLI'' and ``-COVID'' is that the first measures insurance
+  claims related to symptoms
+  consistent with COVID while the second requires a medical diagnosis (through
+  testing or presumed positive by the medical provider). The ``DV'' modifier
+  corresponds to different claims processors (not Change Healthcare) who wish to
+  remain anonymous.
+
+  TODO Do we want to alter the text at all?
```

Comment:
It seems to me that what we have in the Signals and Locations subsection should be sufficient.  That said, in re-reading it, I don't think we cite the Google data (sorry if it's there and I missed it).  Google suggests ([here](https://github.com/GoogleCloudPlatform/covid-19-open-data)) that the data be cited as follows:

@article{Wahltinez2020,
  author = "O. Wahltinez and others",
  year = 2020,
  title = "COVID-19 Open-Data: curating a fine-grained, global-scale data repository for SARS-CoV-2",
  note = "Work in progress",
  url = {https://goo.gle/covid-19-open-data},
}